node/api/renterhost_test.go

```diff
@@ -199,11 +216,34 @@ func TestHostAndRentVanilla(t *testing.T) {
 	}
 
 	// Check the host, who should now be reporting file contracts.
-	//
-	// TODO: Switch to using an API call.
-	obligations := st.host.StorageObligations()
-	if len(obligations) != 1 {
-		t.Error("Host has wrong number of obligations:", len(obligations))
+	var cts ContractInfoGET
+	err = st.getAPI("/host/contracts", &cts)
```

Comment:
Since we are adding a new endpoint, I'd like to see `node/api/client/host.go` extended to support the `/host/contracts` endpoint by adding a `HostContractsGet` method.

@lukechampine @DavidVorick do you think we should move this test to the `siatest` package while we're at it instead of extending an existing test? That would implicitly confirm that `HostContractsGet` is working.